internal/website/content/howto/runtimervar/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Runtime Configuration"
+date: 2019-06-18T14:51:12-07:00
+showInSidenav: true
+---
+
+Working with runtime variables using the Go CDK takes two steps:
+
+1. Open the variable with the `runtimevar` provider of your choice.
+2. As many times as needed, use the `Latest` or `Watch` methods to fetch the
```

Comment:
This is not quite right for `Watch`. You would either use `Latest` as many times as needed, OR you would use `Watch`, probably once in a loop.

I would consider just focusing on `Latest` here, and somewhere down below mention that instead of `Latest`, you can use `Watch` to get edge-based notifications.

Reply:
Removed